Makefile

```diff
@@ -17,16 +17,32 @@ bats-tests: ## Run Helm chart bats tests.
 control-plane-dev: ## Build consul-k8s-control-plane binary.
 	@$(SHELL) $(CURDIR)/control-plane/build-support/scripts/build-local.sh -o $(GOOS) -a $(GOARCH)
 
-control-plane-dev-docker: ## Build consul-k8s-control-plane dev Docker image.
+control-plane-dev-docker: get-dev-image-env ## Build consul-k8s-control-plane dev Docker image.
```

Comment:
Are you missing a `get-dev-image-env` target?

Reply:
oh oops! good catch - this was leftover from an experiment I tried

Reply:
removed!